README.md

```diff
@@ -25,6 +25,8 @@ rails server
 cp .env.example .env # altere as chaves caso queira trocar a aplicação
 docker-compose build
 docker-compose run --rm web rake db:setup
+docker-compose run --rm web rake db:migrate
```

Comment:
When I tried that using rake with multiple parameters it simply didn't work. It ran the first one and ignored the following.

I tried using `'` for the whole command and only for the params but none worked.